build/scripts/main.js

```diff
@@ -21,10 +21,32 @@ Load the time and populate the DOM.
 function LoadTime()
 {
     var timeCurrent = new Date();
+    // Hour
     var timeHours = timeCurrent.getHours();
+    var timeHourPeriod = "AM";
+
+    if (timeHours >= 12)
```

Comment:
I don't like all the if statements. It's longer than it needs to be. Try
if (timeHours >= 12){
    timeHourPeriod = "PM";
}
timeHours = timeHours % 12;

if(timeHours == 0) {
timeHours = 12;
}

Reply:
I like that.